package.json

```diff
@@ -14,10 +14,7 @@
   "bugs": {
     "url": "https://github.com/CSSLint/csslint/issues"
   },
-  "license": {
-    "type": "MIT",
-    "url": "https://github.com/CSSLint/csslint/blob/master/LICENSE"
-  },
+  "license": "SEE LICENSE IN LICENSE",
```

Comment:
I didn't think about it. I just saw that `./LICENSE` existed, so I assumed this was the intention. Usually, MIT is not valid as a license identifier because it has project-specific content.